https://github.com/appneta/tcpreplay/pull/714

From bd93776c50ec1d0574c72f253095afe22eb6bf73 Mon Sep 17 00:00:00 2001
From: Sam James <sam@gentoo.org>
Date: Sun, 13 Feb 2022 16:09:23 +0000
Subject: [PATCH] build: fix bashism in configure

configure scripts are run with /bin/sh which is supposed
to be a POSIX-compliant shell, so if /bin/sh is provided
by e.g. dash (like on Debian) rather than bash, we hit
errors.

This has no effect on functionality for bash &
retains compatibility.

Signed-off-by: Sam James <sam@gentoo.org>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 11a1e4ea..4c577c42 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1808,7 +1808,7 @@ fi
 dnl No 'make test' when cross compile
 
 AC_MSG_CHECKING(for 'make test' profile)
-if test "$host" == "$build" ; then
+if test "$host" = "$build" ; then
     AC_MSG_RESULT([yes])
 else    
     AC_MSG_WARN(Unable to do tests when cross-compiling)

